spec/controllers/orders_controller_spec.rb

```diff
@@ -734,9 +734,9 @@ def setup_past_reservation
               expect(assigns[:order].order_details).to all(be_canceled)
             end
 
-            it "sets the canceled time on the reservation" do
-              assigns[:order].order_details.all? { |od| expect(od.reservation.canceled_at).not_to be_nil }
-              expect(reservation.reload.canceled_at).not_to be_nil
+            it "sets the canceled time on the order_detail" do
+              assigns[:order].order_details.all? { |od| expect(od.canceled_at).not_to be_nil }
+              expect(reservation.order_detail.reload.canceled_at).not_to be_nil
```

Comment:
It's a pain to see in github, so I'm not looking too closely, but if we have direct access to `order_detail` as a variable in specs, I'd lean towards using that over `reservation.order_detail`.